### PR TITLE
Public draft of paper on lessons learned

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by cozzini** [comment]
Hello Greg,
  is there still time to review the work done and propose some 
modifications ?
  Do you have any idea when you think to submit the paper ?

  I am actually going for a short vacation and I was considering reading 
and commenting the printed version of paper and then send back some 
comment to you by the end of the next week.
  Does it make sense this ?

  thanks
  Stefano

On 6/20/13 3:59 PM, Greg Wilson wrote:

> ---
> 
> ```
>     You can merge this Pull Request by running
> ```
> 
>    git pull https://github.com/gvwilson/website blog-lessons-learned
> 
> Or view, comment on, or merge it at:
> 
> https://github.com/swcarpentry/website/pull/617
> 
> ```
>     Commit Summary
> ```
> - Public draft of paper on lessons learned
>   
>   ```
>   File Changes
>   ```
> - _A_ blog/2013/06/lessons-learned.html
>   https://github.com/swcarpentry/website/pull/617/files#diff-0 (269)
>   
>   ```
>   Patch Links:
>   ```
> - https://github.com/swcarpentry/website/pull/617.patch
> - https://github.com/swcarpentry/website/pull/617.diff

## 

Stefano Cozzini  (cozzini@democritos.it)
CNR/IOM uos Democritos c/o SISSA via Bonomea 265 34136 Trieste
tel 040-3787508 fax 040-3787528

**Comment by gvwilson** [comment]
Absolutely --- I won't be submitting until mid-July at the earliest.
Thanks,
Greg

File: blog/2013/06/lessons-learned.html [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+<blockquote>
+<p><em>Edit This</em></p>
+<p>And while it may seem like a trivial thing, editing text is always harder than we expect. We don't want to encourage people to use naive editors like Notepad, and the two most popular legacy editors on Unix (Vi and Emacs) are both usability nightmares. We now recommend a collection of open and almost-open GUI editors, but it remains a stumbling block.</p>
+</blockquote>
+<h3 id="teaching-on-the-web">Teaching on the Web</h3>
+<p>Challenge #5 is to move more of our teaching and follow-up online. We have tried several approaches, from MOOC-style online-only offerings to webcast tutorials and one-to-one online office hours via VoIP and desktop sharing. In all cases, turnout has been mediocre at the start and dropped off rapidly. The fact that this is true of most high-profile MOOCs as well is little comfort...</p>
+<h3 id="waht-vs.-how">Waht vs. How</h3>
```
Comment (ahmadia) [inline blog/2013/06/lessons-learned.html:249]: waht is a typo, right?

File: blog/2013/06/lessons-learned.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+<p>In response, John Reynders (then director of the Advanced Computing Laboratory at Los Alamos National Laboratory) invited the author and Brent Gorda (now at Intel) to teach a week-long course on these topics to LANL staff. The course ran for the first time in July 1998, and was repeated nine times over the next four years. It eventually wound down as the principals moved on to other projects, but several valuable lessons were learned:</p>
+<ol style="list-style-type: decimal">
+<li>Intensive week-long courses are easy to schedule (particularly if instructors are traveling) but by the end of the second day, attendees brains are full and learning drops off significantly.</li>
```
Comment (ahmadia) [inline blog/2013/06/lessons-learned.html:79]: attendee's?

**Pull request merged**